docs/topics/auth/customizing.txt

```diff
@@ -255,13 +255,18 @@ authenticated users do not.
 Do not forget to test for the ``is_active`` attribute of the user in your own
 backend permission methods.
 
+.. _object_permissions:
+
 Handling object permissions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Django's permission framework has a foundation for object permissions, though
 there is no implementation for it in the core. An authentication backend will
 receive the keyword parameter ``obj`` for each object related authorization
-method and can return the object level permission as appropriate.
+method and can return the object level permission as appropriate. When using
+:class:`~django.contrib.auth.mixins.PermissionRequiredMixin` you can implement
+:meth:`~django.contrib.auth.mixins.PermissionRequiredMixin.get_permission_object`
+to check for object permissions.
```

Comment:
I understood this section as explaining object-level permissions in general, not only the backend part. I think it is crucial to explain how this can actually be used.

Reply:
OK: so if we show how to configure the backend, and add the new custom backend, then how about adding this towards the end, showing how it gets used? (Rather than here at the beginning — my thought being that until the custom backend is in place it's not functional/relevant.)

Reply:
I guess you are right: It is probably sufficient to explain how this works with `user.has_perm()` here. I removed the part.